test/Handler/SetOptionTraitTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Horde Log package
+ *
+ * This package is based on Zend_Log from the Zend Framework
+ * (http://framework.zend.com).  Both that package and this
+ * one were written by Mike Naberezny and Chuck Hagenbuch.
+ *
+ * @author     Rafael te Boekhorst <boekhorstb1@b1-systems.de>
+ * @category   Horde
+ * @license    http://www.horde.org/licenses/bsd BSD
+ * @package    Log
+ * @subpackage UnitTests
+ */
+
+namespace Horde\Log\Test\Handler;
+
+use Horde\Log\Handler\MockHandler;
+use PHPUnit\Framework\TestCase;
+use Horde\Log\LogException;
+
+class SetOptionTraitTest extends TestCase
+{
+    public function setUp(): void
+    {
+        $this->mockhandler = new MockHandler();
```

Comment:
Try to not use the mockhandler here. Try this instead:
```
$this->setOptionsTrait = $this->getMockForTrait(SetOptionsTrait::class);
$this->setOptionsTrait->options = new Options();
```

Reply:
ok removed mockhandler from the test